### ISSUE TITLE
X-Requested-With ignores setting

### ISSUE THREAD

**Issue by mtbu** [anthonycr/Lightning-Browser#599]
Lightning seems to sometimes send the 'X-Requested-With' header to the server, regardless of the 'Remove identifying headers' setting in the Privacy options.
Here is a copy of the requests, with 'remove identifying headers' enabled: https://gist.github.com/mtbu/bf12be114e78e92fcb3a847a164ea6d0

**Comment by anthonycr** [comment]
Yes this is a problem, have not found a good workaround. There is an open PR about this, but it does not work as it breaks functionality for pretty much all web browsing https://github.com/anthonycr/Lightning-Browser/pull/403
